core/fileManager/types/type_dev.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "../ifile.h"
+
+#include <filesystem>
+#include <memory>
+
+class TypeDev: public IFile {
```

Comment:
Why another interface ?

Reply:
I thought it will be easier to implement other devices with ITypeDev

Reply:
It just forwards the parent. If needed, we can just create an interface and the childs derive both